README.rst

```diff
@@ -144,7 +158,7 @@ If you want to make your robot speak, you can use the ``Sound.speak`` method:
   from ev3dev2.sound import Sound
 
   sound = Sound()
-  sound.speak('Welcome to the E V 3 dev project!').wait()
+  sound.speak('Welcome to the E V 3 dev project!')
```

Comment:
I started to comment asking why did we remove this, when we still are returning `Popen` objects from `Sound` methods, then noticed we mostly don't anymore. The exception is `beep`, which still needs `.wait()` to make it synchronous. Should it be changed as well?

Reply:
Good point. I'll modify `beep` to match the behavior of the other methods.

Reply:
I've updated `beep` and its derivative `tone`, and in the process fixed the docs example because these methods aren't static anymore.